docs/src/_includes/components/npm_tabs.macro.html

```diff
@@ -0,0 +1,46 @@
+{%- macro npm_tabs(params) -%}
+<div class="c-tabs" data-tabs>
+   <div class="c-tabs__tablist" data-tablist hidden>
+      <button class="c-tabs__tab" data-tab>npm</button>
+      <button class="c-tabs__tab" data-tab>yarn</button>
+      <button class="c-tabs__tab" data-tab>pnpm</button>
+      <button class="c-tabs__tab" data-tab>bun</button>
+   </div>
+   <div class="c-tabs__tabpanel" data-tabpanel>
+      <h3 class="h6 c-tabs__tabpanel__title">npm</h3>
+
+```shell
+{% if params.comment %}# {{ params.comment | safe }}{{ "\n" }}{% endif -%}
+npm {{ conversions.toNpmCommands[params.command] | safe }}{% for arg in params.args %} {{ conversions.toNpmArgs[arg] | safe }}{% endfor %}{% for package in params.packages %} {{ package | safe }}{% endfor %}
+```
+
+   </div>
```

Comment:
```suggestion
    <div class="c-tabs__tabpanel" data-tabpanel>
        <h3 class="h6 c-tabs__tabpanel__title">npm</h3>

        ```shell
        {% if params.comment %}
            # {{ params.comment | safe }}
            {{ "\n" }}
        {% endif -%}
        npm {{ conversions.toNpmCommands[params.command] | safe }}
        {% for arg in params.args %}
            {{ conversions.toNpmArgs[arg] | safe }}
        {% endfor %}
        {% for package in params.packages %}
            {{ package | safe }}
        {% endfor %} ```
    </div>
```
Can we format like this to improve readability?